### ISSUE TITLE
Please add user settings

### ISSUE THREAD

**Issue by ciriousjoker** [iagolaguna/vscodefy#5]
First of all, gj for creating a working extension, the other ones aren't even showing in the statusbar.

Secondly, these are the settings i'd like to see:
show/hide for each control/button
show left/right/center(does center even work?)
show/hide artist

**Comment by iagolaguna** [comment]
Hello @CiriousJoker, thanks for the feedback, it's really important to me.

show/hide for each control/button, I agree if that, I want to convert this extension to TypeScript but before I'm thinking to do this, have more functionalities that do you wanna soon enough?

Show left/right/center(does center even work?), are you talking about the entire StatusBar?
do you wanna specify an alignment to the extension?

When you say  "show/hide artist", do you mean the status of the current track?

**Comment by ciriousjoker** [comment]
yes, I want to be able to move all of the buttons from this extensions to the right (or center if possible).

Currently, only the track title is displayed, but not the artist. I'd like to see a setting to enable that (mby with regex replace to create a pattern in the settings, like vscodify.formatting = "{artist} - {title}")

**Comment by iagolaguna** [comment]
okay, I will go see this, if do you want to contribute to the project let me know, I really need more help to implement new functionalities.

**Comment by ciriousjoker** [comment]
Sorry dude, I'm quite busy with my own project, so I can't really help you with this. Also, the things I mentioned aren't important in any way, they were just meant to be suggestions for future improvement :)

**Comment by iagolaguna** [comment]
oh, relax it's okay when I have enabled time I will do these improvements.
